codility_training/lessons.lesson05.PrefixSums.GenomicRangeQuery/khw970421.js

```diff
@@ -0,0 +1,38 @@
+//정확성은 만족하나 효율성에서 문제
+function solution(S, P, Q) {
+    let k=[];
+    const o=[];
+    const q=['A','C','G','T'];
+    for(let i=0;i<P.length;i++){
+        k=S.split('').slice(P[i], Q[i]+1);
```

Comment:
```suggestion
        const temp = S.substring(P[i], Q[i]+1);
```
split('') => 첫번째 반복
slice('')=> 두번째 반복